src/pyproject_fmt/cli.py

```diff
@@ -11,6 +11,7 @@
 from pathlib import Path
 from typing import Sequence
 
+from pyproject_fmt._version import __version__
```

Comment:
use relative import for this

Reply:
The other import also used an absolute import. I've added 31b5ff1f0d, to also adjust this. Do you agree?